src/components/Congress/CitizenProposeRepealLegislationButton/index.js

```diff
@@ -1,17 +1,17 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import CitizenRepealLegislationModal from '../../Modals/CitizenRepealLegislationModal';
+import Button from '../../Button/Button';
 
 export default function CitizenProposeRepealLegislationButton({ tier, id, section }) {
   const [isModalOpen, setIsModalOpen] = useState(false);
   const handleModalOpen = () => setIsModalOpen(!isModalOpen);
 
   return (
     <>
-      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-      <a href="#" onClick={handleModalOpen}>
+      <Button href="#">
```

Comment:
We need this href here, also onClick is missing?
`<Button onClick={handleModalOpen}>`